bot/seasons/halloween/monster.py

```diff
@@ -0,0 +1,55 @@
+import json
+import logging
+import random
+from pathlib import Path
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path("bot/resources/halloween/monster.json"), "r", encoding="utf8") as f:
+    data = json.load(f)
```

Comment:
Per [PEP 8](https://www.python.org/dev/peps/pep-0008/#constants), module-level constants should be all capital letters.  This could probably use a more informative name than `data` as well.